app-modules/care-team/config/roles/web/care-team_management.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+return [];
```

Comment:
Similar to subscription, let's populate this role with permissions for the `care-team` model, in addition to creating a `CareTeamPolicy`. We can also rename this to move away from the generated default and adhere to a standard casing.

Reply:
@mxm1070 Let's rename this to `care_team_management`